adminutils/adminutils.py

```diff
@@ -189,32 +189,58 @@ async def emoji_add(self, ctx, name: str, url: str, *roles: discord.Role):
         except discord.HTTPException as e:
             await ctx.send(chat.error(_("An error occured on adding an emoji: {}").format(e)))
         else:
+            await ctx.send(_(f"{e} created with the name `{e.name}`."))
```

Comment:
Would be nice if that line contained roles that specified, if they are.

f-strings are not gonna work with i18n, please use `_("{} string {}).format(var, var)`